UnitsNet.Tests/UnitParserTests.cs

```diff
@@ -41,6 +41,23 @@ public void Parse_ReturnsUnitMappedByCustomAbbreviation(string customAbbreviatio
             Assert.Equal(expected, actual);
         }
 
+        [Fact]
+        public void Parse_AbbreviationCaseInsensitive()
+        {
+            var abbreviation = "years";
+            var expected = DurationUnit.Year365;
+            var parser = UnitParser.Default;
+
+            var actual = parser.Parse<DurationUnit>(abbreviation);
+
+            Assert.Equal(expected, actual);
```

Comment:
I would like to advocate [AAA-pattern](https://docs.telerik.com/devtools/justmock/basic-usage/arrange-act-assert).
Basically, one unit test should test one thing, and it should be split in arrange-act-assert parts.

Either split into two tests or consider using `[InlineData]` to pass in parameters with input values and expected value.

Reply:
This is good, but I would suggest dropping the comments. At least for me, when code is divided in 3 paragraphs like this, it is already dead clear what each do. I also omit `// Arrange`, because it's already clear to me that everything before `// Act` is setup code.